src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,121 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
+@SuppressWarnings("unchecked")
 public class ArrayList<T> implements List<T> {
+    private int maxListSize = 10;
```

Comment:
make the `maxListSize` a constant

Reply:
But if I make it a constant, how do I change it when the limit is reached?